packages/react-cool-storage/src/ReactCoolStorageHoc.jsx

```diff
@@ -2,21 +2,12 @@
 import type {ComponentType} from 'react';
 import type {Node} from 'react';
 import type StorageMechanism from './StorageMechanism';
-import type {MessageState} from './ReactCoolStorageMessage';
 
 import React from 'react';
-import InvalidValueMarker from './InvalidValueMarker';
-import ReactCoolStorageMessage from './ReactCoolStorageMessage';
-
-import last from 'unmutable/lib/last';
+import ReactCoolStorageHook from './ReactCoolStorageHook';
 
 type Props = {};
```

Comment:
I don't think this is safe. Won't flow just blat the users prop types when this is applied.

https://flow.org/en/docs/react/hoc/

Reply:
I can't find a way to declare the flow types in such a way that it can deal with the name of the created prop being set via config, in flow versions before or after 0.89.0. I'd rather omit flow types entirely than spend any more time on this, especially as hooks are the preferred method of using this now rather than hocs.

Reply:
Yeah fair call.